Source/KSCrash/Source/KSCrash/Recording/BSG_KSCrashReportFields.h

```diff
@@ -174,6 +174,7 @@
 #define BSG_KSCrashField_Threads "threads"
 #define BSG_KSCrashField_User "user"
 #define BSG_KSCrashField_UserAtCrash "user_atcrash"
+#define BSG_KSCrashField_OnCrash "onCrash"
```

Comment:
Constant name should describe the intent not the value of the constant.

```suggestion
#define BSG_KSCrashField_OnCrashMetadataSectionName "onCrash"
```

Reply:
Renamed constant